android/java/org/chromium/chrome/browser/rewards/BraveRewardsTipConfirmationFragment.java

```diff
@@ -98,7 +98,10 @@ private void initView(
         mNextTipDateText = view.findViewById(R.id.next_tip_date_text);
         mNextTipDateValue = view.findViewById(R.id.next_tip_date_value);
 
-        if (monthlyContribution) {
+        if (status == BraveRewardsSiteBannerActivity.TIP_PENDING) {
+            tipTypeDescriptionTextView.setText(R.string.pending_tip_text);
+            oneTtimeTipNote.setVisibility(View.INVISIBLE);
```

Comment:
Should it be View.GONE?

Reply:
In success case it's visible and for pending case invisible.
That space should be occupied.